quickstart/docker-compose.yml

```diff
@@ -4,106 +4,87 @@ version: "3.9"
+        hard: 65536
     volumes:
-      - data:/usr/share/elasticsearch/data
+      - data:/usr/share/opensearch/data
     ports:
       - 9200:9200
     networks:
-      elastic:
-        ipv4_address: 172.21.0.2
+      - opensearch-net
 
-  kibana:
-    image: docker.elastic.co/kibana/kibana:7.10.2
-    container_name: kibana
+  dashboards:
+    image: public.ecr.aws/opensearchproject/opensearch-dashboards:2.6.0
+    container_name: dashboards
     ports:
       - 5601:5601
     environment:
-      ELASTICSEARCH_URL: http://elasticsearch:9200
-      ELASTICSEARCH_HOSTS: http://elasticsearch:9200
+      - 'OPENSEARCH_HOSTS=["http://opensearch:9200"]'
+      - "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true" # disables security dashboards plugin in OpenSearch Dashboards
     networks:
-      elastic:
-        ipv4_address: 172.21.0.3
+      - opensearch-net
     depends_on:
-      - elasticsearch
-
-  zookeeper:
-    image: wurstmeister/zookeeper
-    hostname: zookeeper
-    container_name: zookeeper
-    ports:
-      - 2181:2181
-    networks:
-      elastic:
-        ipv4_address: 172.21.0.4
+      - opensearch
 
   kafka:
-    image: wurstmeister/kafka
+    image: bitnami/kafka:3.4.0
     container_name: kafka
     ports:
       - 9092:9092
     hostname: kafka
     networks:
-      elastic:
-        ipv4_address: 172.21.0.5
+      - opensearch-net
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: kafka
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://172.21.0.5:9092
-      KAFKA_CREATE_TOPICS: "consumer:1:1,producer:1:1,producer_error:1:1,pseudonyms:1:1,sre:1:1"
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      - KAFKA_ENABLE_KRAFT=yes
+      - KAFKA_CFG_NODE_ID=1
+      - KAFKA_CFG_PROCESS_ROLES=broker,controller
+      - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
+      - KAFKA_BROKER_ID=1
+      - KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093
+      - KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://127.0.0.1:9092
+      - KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=1@127.0.0.1:9093
+      - ALLOW_PLAINTEXT_LISTENER=yes
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-    depends_on:
-      - zookeeper
-
-  logstash:
-    image: docker.elastic.co/logstash/logstash:7.10.2
-    container_name: logstash
-    environment:
-      LS_OPTS: "--config.reload.automatic"
-    networks:
-      elastic:
-        ipv4_address: 172.21.0.6
-    volumes:
-      - ./logstash.conf:/usr/share/logstash/pipeline/logstash.conf
-    depends_on:
-      - kafka
-      - elasticsearch
 
   logprep:
     build:
       context: ..
     image: logprep
     container_name: logprep
     networks:
-      elastic:
-        ipv4_address: 172.21.0.7
+      - opensearch-net
     profiles:
       - logprep
     depends_on:
       - kafka
+    volumes:
+      - ./exampledata/:/home/logprep/
+    entrypoint:
+      - logprep
+      - /home/logprep/config/pipeline.yml
```

Comment:
as the path inside the container doesn't start at quickstart anymore the config cannot resolve correctly

![image](https://user-images.githubusercontent.com/14598229/233987434-9d164807-2b6c-496a-8c67-896f6e48bcb7.png)

Would be the easiest to change the mount and the entrypoint that it will map and resolve correctly.

Reply:
with Logprep not startet in the container from correct location it works.